api/src/main/java/org/apache/cloudstack/api/command/admin/direct/download/RevokeTemplateDirectDownloadCertificateCmd.java

```diff
@@ -54,35 +57,54 @@
     private static final Logger LOG = Logger.getLogger(RevokeTemplateDirectDownloadCertificateCmd.class);
     public static final String APINAME = "revokeTemplateDirectDownloadCertificate";
 
-    @Parameter(name = ApiConstants.NAME, type = BaseCmd.CommandType.STRING, required = true,
-            description = "alias of the SSL certificate")
+    @Parameter(name = ApiConstants.ID, type = CommandType.UUID, required = true,
+            entityType = DirectDownloadCertificateResponse.class,
+            description = "id of the certificate")
+    private Long certificateId;
```

Comment:
@nvazquez 
can `id` be changed to optional ?
it is ok if one of `id` or `name` is passed, right ?

Reply:
Ok @weizhouapache - to keep the previous compatibility name and hypervisor are kept as well